docs/pages/application-access/guides/tcp.mdx

```diff
@@ -0,0 +1,87 @@
+---
+title: Use Teleport for accessing plain TCP apps
+description: How to configure Teleport for accessing plain TCP apps.
+---
+
+# TCP Apps Access (Preview)
+
+Teleport can provide access to any TCP-based application. This allows users to
+connect to applications which Teleport doesn't natively support such as SMTP
+servers or databases not yet natively supported in Database Access.
+
+## Prerequisites
+
+You will need a running Teleport cluster, either self-hosted or in Teleport Cloud.
```

Comment:
I think we should use a similar format as the Application Access Getting Started guide for the Prerequisites (http://localhost:3000/docs/application-access/getting-started/#prerequisites), with a separate step for installing and starting Teleport on the host where the App Service will run. It doesn't add much length to the guide and saves the user from needing to switch context. 

This will be especially useful for cases where a user is only interested in plain TCP apps, not web applications.